doc/quick-start-yaml.adoc

```diff
@@ -5,6 +5,7 @@
 [source,console]
 ----
 $ kubectl apply -f https://github.com/kadalu/kadalu/releases/latest/download/kadalu-operator.yaml
+$ kubectl apply -f https://github.com/kadalu/kadalu/releases/latest/download/csi-nodeplugin.yaml
```

Comment:
LGTM, i think having a note here about why this is decoupled wrt upgrade and node drain would be good.